org.zaproxy.ZAP.json

```diff
@@ -0,0 +1,59 @@
+{
+    "id": "org.zaproxy.ZAP",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "zap",
+    "separate-locales": false,
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk11" 
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--share=network",
+        "--env=JAVA_HOME=/app/jre",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "openjdk",
+            "buildsystem": "simple",
+            "build-commands": [ "/usr/lib/sdk/openjdk11/install.sh" ]
+        },
+        {
+            "name": "zap",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir /app/bin",
+                "ln -s /app/jre/bin/java /app/bin/java",
+                "install -Dm644 org.zaproxy.ZAP.png /app/share/icons/hicolor/128x128/apps/org.zaproxy.ZAP.png",
+                "install -Dm644 org.zaproxy.ZAP.desktop /app/share/applications/org.zaproxy.ZAP.desktop",
+                "install -Dm644 org.zaproxy.ZAP.appdata.xml /app/share/appdata/org.zaproxy.ZAP.appdata.xml",
+                "mkdir /app/share/owasp-zap",
+                "rm org.zaproxy.ZAP.png org.zaproxy.ZAP.desktop org.zaproxy.ZAP.appdata.xml",
+                "cp -r ./* /app/share/owasp-zap",
+                "ln -s /app/share/owasp-zap/zap.sh /app/bin/zap"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/zaproxy/zaproxy/releases/download/v2.8.1/ZAP_2.8.1_Linux.tar.gz",
+                    "sha256": "35fb3924d68bd8d201732a291b888c7f65e15c8e357458882550486d40102431"
+                },
+                {
+                    "type": "file",
+                    "path": "org.zaproxy.ZAP.png"
+                },
+                {
+                    "type": "file",
+                    "path": "org.zaproxy.ZAP.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "org.zaproxy.ZAP.appdata.xml"
```

Comment:
Ideally these files should be maintained by upstream team. (FYI @psiinon)

Reply:
I believe the guys are happy for this to be the case, but they (and I) will need direction on how to maintain upstream. Longer conversation for another time, maybe?

Reply:
Sounds good, just please follow up on that. :+1: